src/components/GlobalConfigs/Share/PermaLink.vue

```diff
@@ -3,15 +3,15 @@
     <v-dialog v-model="dialog" max-width="600px">
       <template v-slot:activator="{ on, attrs }">
         <v-btn
-          rounded
           v-bind="attrs"
           v-on="on"
-          @click="createPermaLink"
-          class="permalink-button text-none font-weight-bold pl-3 pr-3"
+          min-width="34px"
+          width="34px"
           height="34px"
+          class="rounded-circle font-weight-bold"
+          @click="createPermaLink"
         >
-          <v-icon class="share-icon" left> mdi-link-variant </v-icon>
-          <span class="share-text">{{ $t("Share") }}</span>
+          <v-icon class="share-icon"> mdi-share </v-icon>
```

Comment:
Can we include the "Share" text back, but keep it hidden (`v-show="false"` or other CSS)? This is needed for accessibility/screen readers